databases/week2/3School_query.sql

```diff
@@ -0,0 +1,22 @@
+use School;
+CREATE TABLE `Class` (
```

Comment:
its always good practice to use small letter for database/tables and column names